server/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "2"
+
+services:
+
+  app:
+    build: ./
+    volumes:
+      - ./src:/opt/app/src
+      - ./migrations:/opt/app/migrations
+      - ./seeders:/opt/app/seeders
+      - ./config:/opt/app/config
+    environment:
+      - AUTH0_SECRET=<change-me>
+      - SLACK_WEBHOOK_URL=<change-me>
```

Comment:
Is it possible to use local ENV variables instead of manual editing of these variables?

Reply:
what do you mean local ENV?
these variables will appear in process.env like process.env.AUTH0_SECRET @OAGr

Reply:
why do they say <change-me>? Is that supposed to be modified directly to the file?

Reply:
I meant how the local env variables that the user who is running this would use, in order to get the variables to be understood by Docker.

Reply:
if you want some environment var to be defined in a docker container you should set it in .yml file. But may be you want something like this `environment:
    - HAPI_DB_CONNECT=${NODE_DB_CONNECT}` ? (haven't check it) In this case local env NODE_DB_CONNECT value should be used https://stackoverflow.com/questions/30494050/how-do-i-pass-environment-variables-to-docker-containers